java/src/main/java/lesson03/README.md

```diff
@@ -188,27 +180,24 @@ Now change the `FormatterResource` handler method to use `startServerSpan`:
 ```java
 @GET
 public String format(@QueryParam("helloTo") String helloTo, @Context HttpHeaders httpHeaders) {
-    try (Scope scope = Tracing.startServerSpan(tracer, httpHeaders, "format")) {
+    try (Scope scope = startServerSpan(tracer, httpHeaders, "format")) {
         String helloStr = String.format("Hello, %s!", helloTo);
         scope.span().log(ImmutableMap.of("event", "string-format", "value", helloStr));
         return helloStr;
     }
 }
 ```
 
+#### Apply the same to the Publisher
+
+Now, just apply the same changes to the publisher.
+
 ### Take It For a Spin
 
 As before, first run the `formatter` and `publisher` apps in separate terminals.
 Then run `lesson03.exercise.Hello`. You should see the outputs like this:
 
 ```
-# client
```

Comment:
I switched the order here, so that it's clear that the client should be executed *last*